tests/drivers/flash/flash_rpc/Kconfig

```diff
@@ -15,4 +15,5 @@ config APP_INCLUDE_REMOTE_IMAGE
 
 config APP_REMOTE_BOARD
```

Comment:
I'm a bit uncertain if this is still needed

Reply:
not needed anymore